yoti_python_sdk/tests/conftest.py

```diff
@@ -53,5 +70,5 @@ def x_yoti_auth_digest_get():
 
 @pytest.fixture(scope='module')
 def x_yoti_auth_digest_post():
-    with open(AUTH_DIGEST_POST_FILE_PATH, 'r') as auth_digest_file:
+    with io.open(AUTH_DIGEST_POST_FILE_PATH, mode='r', encoding='utf-8') as auth_digest_file:
```

Comment:
This was the issue we experienced where the test was failing when using Python 2.7: the encoding was defaulting to something else, which meant the digest it was comparing in the test was different. `io.open` is needed to ensure compatibility with 2.7